Library/Compiler.php

```diff
@@ -168,9 +166,8 @@ protected function checkRequires()
 
     /**
      * Adds a function to the function definitions
-     *
      * @param FunctionDefinition $func
-     * @param array $statement
+     * @param null $statement
```

Comment:
i am used to see what's the default parameter, but if you agree i will change it back to array